### PR TITLE
+

### PR DESCRIPTION
**Pull request by AndriZhok**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,48 @@
...
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(FrontendDeveloper, BackendDeveloper):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["Python", "SQL", "Django", "JavaScript", "HTML", "CSS"]
```
Comment (Dimosphen1) [inline app/main.py:43]: Do not define `__init__` method here, you should rely on the implementation of the parent classes

File: app/main.py [hunk 2]
```diff
@@ -1 +1,48 @@
...
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["Python", "SQL", "Django"]
```
Comment (Dimosphen1) [inline app/main.py:23]: Do not redefine `self.skills` atribute, extend the list defined in the `self.skills` attribute of the `SoftwareEngineer` class. Apply this change in all related classes

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

File: app/main.py [hunk 3]
```diff
@@ -1 +1,47 @@
...
+        super().__init__(name)
+        self.skills.extend(["Java", "Android studio"])
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(FrontendDeveloper, BackendDeveloper):
+    def __init__(self, name: str) -> None:
```
Comment (pavlopro) [inline app/main.py:41]: Please remove this `__init__` method. It is redundant here.

**Review by pavlopro — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 4]
```diff
@@ -1 +1,46 @@
...
+        super().__init__(name)
+        self.skills.extend(["Java", "Android studio"])
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(FrontendDeveloper, BackendDeveloper):
+    pass
```
Comment (pavlopro) [inline app/main.py:41]: There is no need `pass` here. Remove it.

**Review by pavlopro — CHANGES_REQUESTED** [review]

**Review by pavlopro — APPROVED** [review]
GJ!

**Pull request closed without merging**